### PR TITLE
Updated version of spark notebook images

### PR DESCRIPTION
**Pull request by roskoN**

### CODE REVIEW

File: notebooks/applications/spark/README.md [hunk 1]
```diff
@@ -90,43 +90,6 @@ Steps to create a notebook server are found [here](https://github.com/KubeSoup/d
...
-
-    ```
-    spec:
-      template:
-        metadata:
-          annotations:
-            traffic.sidecar.istio.io/excludeInboundPorts: "7078"
-    ```
-
-
```
Comment (salil-at) [inline notebooks/applications/spark/README.md:129]: I am not sure about this part, as it helps the users to debug if the spark automation is setup correctly.

Comment (roskoN) [reply notebooks/applications/spark/README.md:129]: Hey Salil,

I generally agree that it help debugging issues.

However, a user shouldn't need to debug issues. Spark should just work. If it doesn't work, one of us would probably debug it using Lens and NOT kubectl. :D

Furthermore, this part is automated in a way that is not visible to the user. I don't see why the user should be able to debug it.

This is at least my motivation. If you would like to keep it for other purposes, I could put this in the internal docs.

What do you think?

Comment (salil-at) [reply notebooks/applications/spark/README.md:129]: Yes, keeping it on internal docs is also nice. I agree.

**Review by salil-at — CHANGES_REQUESTED** [review]
```python
os.environ['PYSPARK_SUBMIT_ARGS'] = '--packages "io.delta:delta-core_2.12:1.1.0,org.apache.hadoop:hadoop-aws:3.3.1" pyspark-shell'
```
delta core version needs to be updated

**Review by salil-at — APPROVED** [review]

**Pull request merged**